core/templates/dev/head/pages/exploration-editor-page/services/learner-answer-details-data.service.spec.ts

```diff
@@ -86,19 +75,63 @@ describe('Learner answer details service', function() {
     $httpBackend.verifyNoOutstandingRequest();
   });
 
-  it('should successfully fetch learner answer info data from the backend',
+  describe(
+    'should successfully fetch learner answer info data from the backend',
     function() {
-      var successHandler = jasmine.createSpy('success');
-      var failHandler = jasmine.createSpy('fail');
+      beforeEach(function() {
+        sampleDataResults = {
+          learner_answer_info_data: [{
+            state_name: 'fakeStateName',
+            interaction_id: 'fakeInteractionId',
+            customization_args: 'fakeCustomizationArgs',
+          }]
+        };
+      });
 
-      $httpBackend.expect('GET', '/learneranswerinfohandler/' +
-        'learner_answer_details/exploration/12345').respond(sampleDataResults);
-      LearnerAnswerDetailsDataService.fetchLearnerAnswerInfoData().then(
-        successHandler, failHandler);
-      $httpBackend.flush();
+      it('if it has all the information', function() {
```

Comment:
test statements should begin with `should or should not`

Reply:
Done